src/platforms/javascript/common/install/cdn.mdx

```diff
@@ -8,38 +8,43 @@ import JsBundleList from "~src/components/jsBundleList";
 
 Sentry supports loading the JavaScript SDK from a CDN. Generally we suggest using our npm package (`@sentry/browser`) instead, as using the CDN can create scenarios where Sentry is unable to load due to networking issues or common extensions like ad blockers. If you _must_ use a CDN, take a look at [loading Sentry lazily with our JS loader](../lazy-load-sentry/), which provides a deferred version of our minified ES5 browser bundle. To see what other bundles are available, see [Available Bundles](#available-bundles) below.
 
+## Performance Bundle
```

Comment:
I'm just thinking out lout here so feel fee to disregard but maybe we should make it clear that this is what we consider the default bundle? Right now it seems like we're not really providing a quick and easy "this is the bundle you most probably wanna use" code snippet. But people actually have to go in and figure out which one of the three they need. 

```suggestion
## Base Bundle
```

Reply:
Isn't the one with Replay the one we want users to use?

Downgrade from there if they don't want a feature for the sake of a smaller bundle size?

Reply:
Yeah, I guess that's a general question to solve 😅 IMHO for now, especially while replay is still in beta, it makes sense to not have it the default bundle? I'd call it "Default bundle" maybe, as Base seems to me like it is the "minimum" bundle?

Reply:
> IMHO for now, especially while replay is still in beta, it makes sense to not have it the default bundle? 

Agreed. Also, the bundle size footprint should be considered. Not saying we shouldn't promote it as the default bundle in the future but let's discuss this when it's stable.

> I'd call it "Default bundle" 

Sure, sounds also good to me 👍

Reply:
Agreed, thanks folks